### DISCUSSION TITLE
Tuần 2+3, tìm chủ đề, tìm câu, và làm mở bài

### DISCUSSION THREAD

**Discussion by dangxuantruong10a13** [dangxuantruong10a13/nhom9_10a13#27] [General]
Mọi người ơi tuần này mỗi người sẽ tìm ra 2 chủ đề xung quanh cuộc sống (Hay nhất đối với bản thân). Sau khi tìm xong cả nhóm sẽ vote ra chủ đề hay nhất rồi đi tìm câu ca dao thành ngữ tục ngữ,... cho chủ đề đó, mỗi người 2 câu (Hay nhất đối với bản thân), tự làm càng tốt, không thì google nha. Xong hết thì đến phân công mở bài và làm word, mở bài sẽ do Lan Anh, Hân bàn nhau rồi làm nha ai cãi không aisshhshhshse chket tiệt. Làm word sẽ tùy mà phân công sau

**Comment by dangxuantruong10a13** [comment]
À Lâm tạo dùm project với, đang xài điện thoại. Mọi người vào project tự điền tên vào nhé tui phân công trong đó

**Reply by TranBaoLam10C13** [reply]
Um

**Comment by TranBaoLam10C13** [comment]
Chủ đề: 1. Hạnh phúc gia đình
              2. Tương lai tươi sáng

**Reply by dangxuantruong10a13** [reply]
Ngon

**Reply by TranBaoLam10C13** [reply]
Ê z là ca dao, tục nghĩ với bài văn à

**Reply by TranBaoLam10C13** [reply]
Tục ngữ *

**Reply by dangxuantruong10a13** [reply]
Là mình làm câu dao tục ngữ theo chủ đề xong phân tích thành bài văn ák

**Reply by dangxuantruong10a13** [reply]
Phải hong hay tui lộn

**Comment by 04lananh** [comment]
Theo t nghĩ thì tụi mik lm cđ là sự tử tế trong cuộc sống sẽ hay hơn á
Còn mng có ý tưởng j nx ko

**Reply by dangxuantruong10a13** [reply]
Kkkkk xong hết r mình vote

**Reply by NgHan12pn** [reply]
theo @04lananh là cách đối nhân xử thế trong cuộc sống đúng hong

**Reply by 04lananh** [reply]
Ukm thì kiểu như là mình cảm nhận như thế nào về cách mng đối xử với bản thân mình khi mình lịch sự, tử tế với họ
Kiểu là tình thương giữa người với người á

**Comment by TangVinhAn-PNc13** [comment]
theo tui nghĩ là làm chủ đề giống lan anh khá hay ó

**Reply by dangxuantruong10a13** [reply]
Ngon

**Comment by NgHan12pn** [comment]
ủa z là chốt chủ đề của @04lananh chọn hả mng

**Reply by dangxuantruong10a13** [reply]
bà muốn thêm chủ đề nào nữa hong

**Reply by dangxuantruong10a13** [reply]
oke tui cũng theo kkkkk

**Comment by dangxuantruong10a13** [comment]
rầu mở vote nek

**Reply by dangxuantruong10a13** [reply]
ai theo Lan Anh bấm 1, theo Lâm chủ đề nào viết ra r viết số 2 nha nha nha, Lâm chọn Lan Anh cũng đc o,o,o,o,o

**Reply by dangxuantruong10a13** [reply]
còn hong thì tìm 2 chủ đề mình thấy hay nhất nhaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

**Reply by TranBaoLam10C13** [reply]
theo lan anh đi

**Reply by NgHan12pn** [reply]
thế làm mấy chủ đề

**Reply by NgHan12pn** [reply]
thấy chủ đề của lan anh đề xuất ổn r đó

**Reply by dangxuantruong10a13** [reply]
oke giờ sẽ chọn câu ca dao tục ngữ thành ngữ,.alksdjglsadgj bla bla bla về chủ đề đối xử tử tế nkaka kakakakaakakkakaa

**Reply by 04lananh** [reply]
Kim vàng ai nỡ uốn câu
Người khôn ai nỡ nói nhau nặng lời

**Reply by 04lananh** [reply]
Mở rộng bằng câu nói này cx hay nè 
Lòng tốt là ngôn ngữ mà người điếc có thể nghe và người mù có thể nhìn thấy. - Mark Twain

**Reply by 04lananh** [reply]
Hoặc là câu 
Sự tử tế, dù nhỏ tới thế nào, không bao giờ là lãng phí-  Aesop

**Reply by 04lananh** [reply]
Mng có câu nào nx ko??

**Reply by dangxuantruong10a13** [reply]
có ai có thơ về cái này hongggjlasdkgjsn

**Comment by 04lananh** [comment]
Mng ơi h mik phân việc luôn ik

**Reply by 04lananh** [reply]
H chia như vầy nè
1 ng làm mở bài + giải thích (lý lẽ)
1 ng làm biểu hiện + dẫn chứng
1 ng làm biện pháp + phê phán + ý nghĩa + mở rộng ... ( tất cả các ý này là 1 đoạn nha mng tách ra cx đc nói chung là tóm gọn lại á chứ nó ko dài đâu)
1 ng làm bản thân tôi + kết bài 
1 người tổng hợp làm pwp nha

**Reply by 04lananh** [reply]
T thấy là chia như vầy hợp lý rồi nè
Mng chọn phần rồi làm nha

**Reply by NgHan12pn** [reply]
ủa này làm word hay pwp

**Reply by 04lananh** [reply]
Này làm word nha t ghi lộn sogy🥲🥲

**Reply by dangxuantruong10a13** [reply]
còn lâu mà giờ chăm chút mở bài trước ik kkkkkkkkkkkkk 2 tuần tới làm thân bài đc r

**Reply by TranBaoLam10C13** [reply]
tui làm  làm biểu hiện + dẫn chứng nha

**Reply by 04lananh** [reply]
@dangxuantruong10a13 1 ng làm thôi m ơi
Chia ra làm cho lẹ, mỗi người 1 phần để khỏi bất đồng ý kiến á
Nên theo t thì ng nào lm phần nào thì nên làm hết

**Reply by dangxuantruong10a13** [reply]
ý là tuần này chỉ cần cái mở bài là đc rầu

**Reply by dangxuantruong10a13** [reply]
tại vì đây là bài thực hành word chứ hong phải làm văn nên là mình sẽ 90% cho người người nhà nhà làm word

**Reply by dangxuantruong10a13** [reply]
2 người làm văn là đủ r

**Reply by dangxuantruong10a13** [reply]
hoặc 5 người làm văn xong 2 tuần cuối sẽ nhảy vào làm word

**Reply by dangxuantruong10a13** [reply]
chia việc theo từng đoạn

**Reply by dangxuantruong10a13** [reply]
được hong nek

**Comment by 04lananh** [comment]
Mng đã chốt câu tục ngữ nào chưa hay có đề xuất câu nào ko??

**Reply by NgHan12pn** [reply]
theo tui nghĩ là câu:
   "Lời nói chẳng mất tiền mua
     Lựa lời mà nói cho vừa lòng nhau"

**Comment by 04lananh** [comment]
Ukm để thầy dễ theo dõi tiến trình thì mng làm đc j thì cứ đăng lên phần project nha
Chưa làm cx đăng luôn 🤧🤧🤧

**Comment by 04lananh** [comment]
T làm phần ý nghĩa mở rộng ... nha
Còn đại ca @dangxuantruong10a13 làm j ạ:"vv

**Reply by dangxuantruong10a13** [reply]
![image](https://user-images.githubusercontent.com/91663261/155843524-479bec92-88af-4db5-bc1f-eedf2bc7b811.png)
nhảy lên trên dọc ik đại ka

**Reply by dangxuantruong10a13** [reply]
tui recommend là theo phương án 2 nha

**Reply by NgHan12pn** [reply]
thấy p/á 2 cũng oke đó

**Reply by 04lananh** [reply]
Dạ em vẫn chưa hỉu🤧🤧🤧
Phương án 2 là 5 người lm văn xong r tự đánh word hả

**Reply by dangxuantruong10a13** [reply]
oke vậy tui làm bản thân tui nka

**Reply by 04lananh** [reply]
Xong tự chỉnh luôn hay sao

**Reply by dangxuantruong10a13** [reply]
kkkkkk

**Reply by dangxuantruong10a13** [reply]
là làm xong tự chỉnh đoạn của mình rồi giao nộp cho tui

**Reply by dangxuantruong10a13** [reply]
làm theo kĩ thuật thầy muốn mình làm

**Reply by dangxuantruong10a13** [reply]
hoặc là 5 người làm 1 đoạn 1 tuần làm xong 2 đoạn

**Reply by NgHan12pn** [reply]
ủa khoan nếu mà làm theo p/á 2 thì cái ở trên lan anh phân công 1 ng tổng hợp làm word đâu cần thiết nữa đúng hong

**Reply by dangxuantruong10a13** [reply]
hoặc là 1 người làm xong 1 đoạn rồi 4 người khác vào nhận xét r đi chỉnh word

**Reply by 04lananh** [reply]
Thấy cứ kì kì sao í
T thấy lm vậy thì nó ko đc liên kết lắm

**Reply by dangxuantruong10a13** [reply]
tại seo hong liên kết o,o,o,o,o,o

**Reply by TranBaoLam10C13** [reply]
đừng có them phương án nào nx chốt 1 cái đi

**Reply by dangxuantruong10a13** [reply]
thử tưởng tượng nek, bây giờ tuần này còn 1 ngày nữa sẽ hết, cho Lan Anh mở bài đi.....Mở bài xong 4 người còn lại vào nhận xét "ù ù hay hay hay, còn cái này chưa hay neefk, ù ù ý hay quá ú ú ú" xong hết rồi Lan Anh sẽ chỉnh word theo kỹ thuật thầy đưa, rồi đưa lại cho 4 người còn lại xem rồi giao nộp lên phần code

**Reply by 04lananh** [reply]
Thì mỗi người một kiểu cái nó bị rời rạc á
Người chọn font bày người chỉnh font khác r màu sắc kích thước chữ chưa kể cả cách trình bày nx

**Reply by dangxuantruong10a13** [reply]
người người làm văn nhà nhà làm word lun

**Reply by 04lananh** [reply]
Ko thì 1 người chỉnh hết r cả nhóm ý kiến thôi

**Reply by NgHan12pn** [reply]
theo ý thầy làm z để tất cả mng đều bít cách thực hành làm word thì theo cách của trường nói cũng đúng mà

**Reply by TranBaoLam10C13** [reply]
5 người làm văn xong 2 tuần cuối sẽ nhảy vào làm word , r chia theo lan anh ?

**Reply by 04lananh** [reply]
Hơn nx còn cần ng tổng hợp bài văn lại nx tại sợ có đứa lạc đề á với lại đây là bài nghị luôn nên ý tưởng mỗi người khác nhau, góc nhìn cx khác nhau luôn xong cái tự chỉnh cái thành ra mỗi đứa 1 bài

**Reply by dangxuantruong10a13** [reply]
thì tui có nói là mn sẽ nhận xét é

**Reply by dangxuantruong10a13** [reply]
@TranBaoLam10C13 đổi lại là làm văn xong tự chỉnh word luôn r é kkkkk

**Reply by 04lananh** [reply]
Hay là h như vầy nè 
T mik làm xong bài văn sớm xíu r dành 1 tuần để chỉnh sửa nd sao cho khớp ý với nhau
R tuần cuối thì mng thống nhất các cách định dạng( như là làm font chữ màu sắc các kiểu) r ồi mng tự chỉnh bài của mik r gửi cho đại ka Trường tổng hợp lại r gửi thầy đc ko?

**Reply by NgHan12pn** [reply]
nếu mà @04lananh  làm phần ý nghĩa mở rộng thì tui làm biện pháp với phê phán nha hay phần đó pà làm hết

**Reply by dangxuantruong10a13** [reply]
cũng được nek. Thường thì sẽ thống nhất các định dạng đầu tiên cho đỡ phải chỉnh lại hơn nek. Lan Anh thử nêu ra quy trình làm các tuần thử, từ giờ tới 31/3 sẽ có 4 tuần + thêm 1 ngày tuần này nữa

**Comment by 04lananh** [comment]
Mng ơi hay là 4 người làm nd thui người còn lại sẽ là người tổng hợp bài lại r chỉnh sửa sao cho các ý mà mng làm khớp với nhau để tránh lạc đề

**Reply by dangxuantruong10a13** [reply]
mỗi tuần sẽ làm 1 phần r nhận xét nên là khó lạc đề lắm, tổng hợp k cần thiết lắm lắm lắm lắm, để tui gom lại r đưa thầy là đc r.

**Comment by dangxuantruong10a13** [comment]
vậy ai tìm ra đc câu, thơ nào hay với chủ đề Lan Anh chưa kkkkkk

**Comment by dangxuantruong10a13** [comment]
tui suggest 
Lá rách lá lành đùm bọc lấy nhau
Một cành tươi năm bảy cành sầu
Chim kia còn biết nhớ nhau nữa người..

và 
Nhiễu điều phủ lấy giá gương

Người trong một nước phải thương nhau cùng.

**Reply by 04lananh** [reply]
Tụi mik có thể lấy 1 câu chủ đề còn mấy câu khác có thể thêm ở phần mở rộng, mở bài với kết bài nx á nên là có câu nào hay mng cứ ý kiến nha

**Reply by 04lananh** [reply]
@dangxuantruong10a13 thêm câu nói đc ko m

**Reply by dangxuantruong10a13** [reply]
hong biết nữa thầy k có ghi là tìm câu nói

**Comment by dangxuantruong10a13** [comment]
Rồi oke hong ý kiến j nữa mình sẽ tự giao việc luôn nhé

**Reply by dangxuantruong10a13** [reply]
@04lananh go go pặn ơi thích làm j làm

**Comment by dangxuantruong10a13** [comment]
à các bạn muốn up file để nhận xét thì tự up bài trong phần issues nhé cho đỡ phải mò trong đây @04lananh @TranBaoLam10C13 @NgHan12pn @TangVinhAn-PNc13

**Comment by 04lananh** [comment]
H phân công nè
     +Mở bài giao cho @NgHan12pn nha nhớ viết sao cho ngầu ngầu xíu😎😎😎
     +Phần giải thích thì @TangVinhAn-PNc13 nha ( nếu là tục ngữ , ca dao jj đó thì phải giải thích cả nghĩa đen lẫn  nghĩa bóng nhảooif tại sao phải sống tử tế bla bla ... miễn hay là đc😆
    + Biểu hiện, dẫn chứng thì @TranBaoLam10C13    chịu trách nhiệm nha ( dẫn chứng thì nêu những tấm gương tử tế xung quanh ta á ... phần này thì bn Lâm là vij pro r nhỉ😉😉😉)
    + T sẽ làm phần phê phán, ý nghĩa rồi mở rộng nha ( mng cứ tin ở tui😳😳😳)
    + Cuối cùng là phần bản thân tôi và kết bài sẽ do đại ka @dangxuantruong10a13  gánh vác ( nhớ viết hay hay xíu nha m 😉 chèn thêm mấy câu nói hay tục ngữ ca dao tương tự đó

**Reply by 04lananh** [reply]
R còn ai ý kiến j nx ko ( mà có cx ko cho ý kiến nx âu😳😳😳)

**Reply by dangxuantruong10a13** [reply]
oke nhớ theo chủ đề tử tế hoài nha ae tránh lạc đề sửa lại mệt ỉa

**Reply by dangxuantruong10a13** [reply]
muốn thoáng thời gian thì ae làm 2 phần trong 1 tuần nha. Là tuần sau sẽ làm xong giải thích với dẫn chứng

**Reply by dangxuantruong10a13** [reply]
![image](https://user-images.githubusercontent.com/91663261/155846094-258bd481-d858-4413-981e-c3b1e1b16d94.png)
ngon, còn 3 tuần

**Reply by 04lananh** [reply]
Mng đừng lo lạc đề đã có đại ka Trường gánh

**Reply by dangxuantruong10a13** [reply]
ngon
